src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,321 @@
+        })
+      }
+    ];
+
+    const rowsForParameters = triggerBinding.spec.params.map(
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    return (
+      <div id="triggerBinding">
+        <h1>{triggerbindingName}</h1>
+        <Tabs
+          ariaLabel="listbox"
+          className="some-class"
+          iconDescription="show menu options"
+          role="navigation"
+          tabContentClassName="tab-content"
+        >
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.triggerBinding.details',
+              defaultMessage: 'Details'
+            })}
+            onClick={() => {
+              this.handleTabSelect('details');
+            }}
+            role="presentation"
+            selected={showDetails}
+            tabIndex={0}
+          />
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.triggerBinding.yaml',
+              defaultMessage: 'Yaml'
+            })}
+            onClick={() => {
+              this.handleTabSelect('yaml');
+            }}
+            role="presentation"
+            selected={showYAML}
+            tabIndex={0}
+          />
+        </Tabs>
+        {showDetails && (
+          <div className="details">
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.dateCreated',
+                  defaultMessage: 'Date Created:'
+                })}
+              </span>
+              <FormattedDate
+                date={triggerBinding.metadata.creationTimestamp}
+                relative
+              />
+            </p>
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.labels',
+                  defaultMessage: 'Labels:'
+                })}
+              </span>
+              {formattedLabelsToRender.length === 0
+                ? intl.formatMessage({
+                    id: 'dashboard.triggerBinding.noLabels',
+                    defaultMessage: 'None'
+                  })
+                : formattedLabelsToRender.map(label => (
+                    <Tag type="blue">{label}</Tag>
+                  ))}
+            </p>
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.namespace',
+                  defaultMessage: 'Namespace:'
+                })}
+              </span>
+              {triggerBinding.metadata.namespace}
+            </p>
+            <Table
+              title={intl.formatMessage({
+                id: 'dashboard.triggerBinding.title',
+                defaultMessage: 'Parameters'
+              })}
+              headers={headersForParameters}
+              rows={rowsForParameters}
+              selectedNamespace={selectedNamespace}
+              emptyTextAllNamespaces={intl.formatMessage({
```

Comment:
To ensure consistency between the 2 messages here we should store the result of `intl.formatMessage` in a variable and use it for both props.